src/app/pages/DeclarationForm/styles.scss

```diff
@@ -0,0 +1,351 @@
+							width: 70px;
+							text-align: center;
+						}
+
+						&:nth-child(3) {
+							vertical-align: middle;
+							min-width: 70px;
+							width: 70px;
+							text-align: center;
+						}
+					}
+
+					:is(td:nth-child(2), td:nth-child(3)) {
+						text-align: center;
+						vertical-align: middle;
+					}
+				}
+
+				:is(td, th) {
+					padding: 15px 10px;
+					border: 1px solid #edeff1;
+				}
+
+				tbody td {
+					font-weight: 500;
+				}
+			}
+		}
+
+		.more-info {
+			margin-top: 8px;
+		}
+
+		.captcha {
+			&-title {
+				margin-top: 5px;
+				font-weight: 700;
+				color: $color-blue;
+			}
+
+			&-text {
+				font-size: 1.4rem;
+				color: $color-blue;
+				font-weight: 600;
+				margin-top: 15px;
+			}
+
+			&-code {
+				height: calc(1.5em + 0.5rem + 2px) !important;
+				padding: 0.15rem 0.5rem;
+				color: #393939;
+				width: 200px;
+				margin-top: 20px;
+			}
+		}
+
+		.submit-button-center {
+			text-align: center;
+			button[type='submit'] {
+				all: unset;
+				color: $color-white;
+				background-color: $color-blue;
+				padding: 0.45rem 0.9rem;
+				font-size: 0.9rem;
+				line-height: 1.5;
+				border-radius: 0.15rem;
+				min-width: 100px;
+				width: 30%;
+				margin-bottom: 8px;
+			}
+		}
+
+		.type-of-test-object {
+			display: flex;
+			&-title {
+				margin: 0;
+			}
+			label {
+				display: flex;
+				margin-right: 10px;
+				margin-bottom: 10px;
+				margin-top: 10px;
+				flex-direction: row;
+				align-items: center;
+			}
+
+			input {
+				margin-right: 8px;
+			}
+		}
+
+		.place-of-test-object {
+			margin-bottom: 10px;
+
+			label {
+				display: flex;
+				flex-direction: row;
+				align-items: center;
+				input {
+					margin-right: 8px;
+				}
+			}
+		}
+
+		.place-of-test-object-title {
+			margin: 0;
+		}
+
+		.background-disease {
+			margin-bottom: 10px;
+			display: flex;
+
+			&-title {
+				margin: 0;
+			}
+			label {
+				display: flex;
+				flex-direction: row;
+				align-items: center;
+				margin-right: 10px;
+				input {
+					margin-right: 8px;
+				}
+			}
+		}
+
+		.type-of-background-disease {
+			margin-bottom: 10px;
+			&-title {
+				margin: 0;
+			}
+			label {
+				display: flex;
+				flex-direction: row;
+				align-items: center;
+			}
+
+			input {
+				margin-right: 8px;
+			}
+		}
+
+		.is-used-molnupiravir {
+			display: flex;
+			flex-direction: row;
+			&-title {
+				margin: 0;
+			}
+			label {
+				display: flex;
+				flex-direction: row;
+				align-items: center;
+				* {
+					margin-right: 8px;
+				}
+			}
+		}
+
+		.used-molnupiravir-title {
+			font-weight: 700;
+			color: $color-blue;
+		}
+
+		.list-symptom-after-used-molnupiravir {
+			display: grid;
+			grid-template-columns: 30% 70%;
+
+			label {
+				display: flex;
+				flex-direction: row;
+				align-items: center;
+				span {
+					width: 100%;
+				}
+
+				input {
+					margin-right: 8px;
+				}
+
+				input[type='text'] {
+					width: 700%;
+				}
+			}
+		}
+	}
+}
```

Comment:
text indent is too large. Recommended text indent is 2 spaces

Reply:
i don't know why code on repo show like that, i have config with eslint and prettier and also githook to but sometime it's still happening (just on repo, my code on local not showing like that -.-)